tests/unit/command/ls/test_ls_colors.py

```diff
@@ -0,0 +1,58 @@
+from dvc.command.ls.ls_colors import LsColors
+
+
+def colorize(ls_colors):
+    def _colorize(f, spec=""):
+        fs_path = {
+            "path": f,
+            "isexec": "e" in spec,
+            "isdir": "d" in spec,
+            "isout": "o" in spec,
+        }
+        return ls_colors.format(fs_path)
+
+    return _colorize
```

Comment:
It'd be one less redirection if we just did following on all of the tests:
```py
ls_colors.format({"path": f, "isexec": True})
```

Reply:
Thank you @skshetry for the point!
The changes came from the comment https://github.com/iterative/dvc/pull/3351#discussion_r382584512 in terms of making tests more readable

Reply:
Are other keys mandatory? If they aren't, I don't really think there's any need for redirection.

If they are, you can just:
```py
shape = {
  "isexec": False,
  "isdir": False,
  "isout": False,
}
ls_colors.format({"path": f, **shape, "isexec": True})
```

But, I'm fine with it as well. But, one less redirection would be great.

Reply:
> Are other keys mandatory?

Nope, there are not. In fact before [the comment](https://github.com/iterative/dvc/pull/3351#discussion_r382584512) my tests [were](https://github.com/iterative/dvc/commit/d4b0194c3b3d0caaa4236e1431a15f39a82e83b8#diff-02572f52f28eecd20a866fbcd2629201R24) as you described

I don't see the problem with additional step here (redirection you mean). I think it makes sense not to add additional functionality related to tests only, but let it keep here cause it's thin enough